python/packages/kagent-langgraph/src/kagent/langgraph/_executor.py

```diff
@@ -171,12 +202,245 @@ async def _stream_graph_events(
                 )
             )
 
+    async def _handle_interrupt(
+        self,
+        interrupt_data: list[Any],
+        task_id: str,
+        context_id: str,
+        event_queue: EventQueue,
+        task_store: TaskStore,
+    ) -> None:
+        """Handle interrupt from LangGraph and convert to A2A input_required event."""
+        # Extract interrupt details
+        # Format: [{"value": {"action_requests": [...], "review_configs": [...]}}]
```

Comment:
Can we use the typed dicts from the sdk? e.g. https://github.com/langchain-ai/langchain/blob/d9e659ca4f020908a360118e7a2a225d86e4b555/libs/langchain_v1/langchain/agents/middleware/human_in_the_loop.py#L39